frontends/web/src/app.tsx

```diff
@@ -181,7 +181,7 @@ class App extends Component<Props, State> {
                         {/* Use with TypeScript: {Route<{ deviceID: string }>({ path: '/manage-backups/:deviceID', component: ManageBackups })} */}
                         {/* ManageBackups and DeviceSwitch need a key to trigger (re-)mounting when devices change, to handle routing */}
                         <ManageBackups
-                            path="/manage-backups/:deviceID"
+                            path="/manage-backups/:deviceID/:sdCardInserted?"
```

Comment:
why put sdCardinserted into the url? seems like the wrong place, especially since 

- you make api calls in BackupsV2 anyway. Why not make an api call there to check if the sd card is inserted?
- you turn the prop into a state and modify it after. That is rarely good, as then you have a useless prop lying around.
- you use ManageBackups for both v1 and v2, but the url param is only used for v2, which is inconsistent.

I'd just do it like in the v1 backups component and check if the sd card is inserted within.

Reply:
The reason I'd done it this way is for UX purposes. In v1, the user has to click the "I have inserted the SD card" button after they have inserted it, and they can click this button even before inserting the SD card, and nothing happens when the button is pressed. I don't think that's great UX so my idea was to make it more seamless. 
This way there is no re-rendering (Preact still uses componentWillUpdate and other deprecated methods and doesn't seem to have getDerivedStateFromProps yet) and the updates get listed automatically when the user presses ok on the device. I agree that the props/state handling in manage backups is not ideal, but maybe a better solution would then be to separate manage backups into two components, one for v1 and one for v2, same as is currently done with backups.